src/main/main.ts

```diff
@@ -147,18 +217,16 @@ const createWindow = async () => {
 
   win.on('resize', () => {
     if (win) {
-      const browserViews = win?.getBrowserViews();
-
       // set BrowserView's bounds explicitly
-      browserViews?.forEach((browserView) => {
+      win?.getBrowserViews()?.forEach((browserView) => {
```

Comment:
No need for optional chaining here after win and highly likely after getBrowserViews()